domain/tutoringModel/learning_path.py

```diff
@@ -61,23 +61,33 @@ def calculate_sequence(self, learning_element_types, input_learning_style):
         return sum
 
     def sort_learning_path(self, learning_path):
-        sort_learning_path = {}
+        print(learning_path)
+        sort_learning_path = []
+        sort_learning_path_dict = {}
+
         sorted_keys = sorted(
             learning_path, key=learning_path.get, reverse=True)
 
         for w in sorted_keys:
-            sort_learning_path[w] = learning_path[w]
-
+            sort_learning_path_dict[w] = learning_path[w]
+
+        print(sort_learning_path_dict)
+        for _ in range(len(learning_path)):
+            test = max(learning_path, key=learning_path.get)
```

Comment:
The variable "test" is unclear, it's a key or a learning_elements

Reply:
renamed